rotkehlchen/typing.py

```diff
@@ -366,6 +368,17 @@ class Location(DBEnumMixIn):
     BISQ = 35
 
 
+class DetailedLocation(NamedTuple):
```

Comment:
What's this? It's not used anywhere. Was that also something you wanted to move out?

Reply:
I wanted to use something instead of the location_label, but the location_label I believe is the best solution